CRM/Contact/Page/View/Relationship.php

```diff
@@ -68,6 +70,15 @@ public function view() {
     $relationship = new CRM_Contact_DAO_Relationship();
     $relationship->id = $viewRelationship[$this->getEntityId()]['id'];
 
+    $viewRelationship[$this->getEntityId()]['name'] = CRM_Contact_BAO_Relationship::formatContactName([
+      'name' => $viewRelationship[$this->getEntityId()]['name'],
+      'is_deceased' => Contact::get(FALSE)
+        ->addWhere('id', '=', $viewRelationship[$this->getEntityId()]['cid'])
+        ->addSelect('is_deceased')
+        ->execute()
+        ->first()['is_deceased']
```

Comment:
@mattwire need a `,` at the end of this line

Reply:
Thanks @seamuslee001